errors.go

```diff
@@ -2,9 +2,10 @@ package cloudflare
 
 // Error messages
 const (
-	errEmptyCredentials = "invalid credentials: key & email must not be empty"
-	errMakeRequestError = "error from makeRequest"
-	errUnmarshalError   = "error unmarshalling the JSON response"
+	errEmptyCredentials     = "invalid credentials: key & email must not be empty"
+	errMakeRequestError     = "error from makeRequest"
+	errUnmarshalError       = "error unmarshalling the JSON response"
+	errRequestNotSuccessful = "Error reported by API"
```

Comment:
Nitpick: In order to match the established pattern, the error message should start with a lower case letter.